CHANGELOG.md

```diff
@@ -2,7 +2,8 @@ Changelog
 =========
 
 # Unreleased (0.21.0)
-* Nothing yet
+* Fix gRPC server stream handling to be compatible with Java gRPC server by closing
+  the send stream post dispatching the client request.
```

Comment:
nit: no need of mentioning `by closing the send stream post dispatching the client request.`